pkg/util/test/util.go

```diff
@@ -1750,6 +1763,150 @@ func ValidateTelemetry(pxImageList map[string]string, cluster *corev1.StorageClu
 	return ValidateTelemetryUninstalled(pxImageList, cluster, timeout, interval)
 }
 
+// ValidateAlertManager validates alertManager components
+func ValidateAlertManager(pxImageList map[string]string, cluster *corev1.StorageCluster, timeout, interval time.Duration) error {
+	if cluster.Spec.Monitoring != nil && cluster.Spec.Monitoring.Prometheus != nil {
+		if cluster.Spec.Monitoring.Prometheus.Enabled {
+			logrus.Infof("Prometheus is enabled")
+			if cluster.Spec.Monitoring.Prometheus.AlertManager.Enabled {
+				logrus.Infof("AlertManager is enabled")
+				return ValidateAlertManagerEnabled(pxImageList, cluster, timeout, interval)
+			}
+			logrus.Infof("Prometheus is not enable, validating AlertManager is not deployed")
+			return ValidateAlertManagerDisabled(pxImageList, cluster, timeout, interval)
+		}
+	}
+
+	logrus.Infof("AlertManager is disabled")
+	return ValidateAlertManagerDisabled(pxImageList, cluster, timeout, interval)
+}
+
+// CreateAlertManagerSecret creates secret required for AlertManager component
+func CreateAlertManagerSecret(secretNamespace string) error {
```

Comment:
Can we create this from a file instead of hard coding it here? Also, we don't need this function. We can use what we do in the migration test - https://github.com/libopenstorage/operator/blob/master/test/integration_test/migration_test.go#L113.

Reply:
Done